### PR TITLE
New article: Let's begin Scala with GitBucket!

### PR DESCRIPTION
**Pull request by takezoe**
This is translation of  nazoking's Japanese article on Qiita: http://qiita.com/nazoking@github/items/4518266ff0f4fd92bd0a

### CODE REVIEW

**Comment by McFoggy** [comment]
@takezoe I have created a PR to provide some fixes & corrections because I didn't wanted to push directly in your branch.
If you think it is easier for maintenance, I can push to your branch.

File: _posts/2017-01-08-lets-begin-scala-with-gitbucket.md [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+
+Put this controller into `src/main/scala/gitbucket/core/IndexController.scala` and open `/hoge` in your browser. You will see `Hello World`.
+
+Twirl recognize parts of after `@` as Scala code and process them in nice touch. Almost template files in `src/twirl/gitbucket/core` are matched with url. Try to see pages you care about on GitBucket.
+
+In Scalatra, you can retreive request parameters as following:
+
+```scala
+  get("/hoge/:aaa") {
+    "aaa="+ parem("aaa")+ " bbb="+parem("bbb")
```
Comment (McFoggy) [inline _posts/2017-01-08-lets-begin-scala-with-gitbucket.md]: You know I am not a scala guru, but are your sure about `parem`?

File: _posts/2017-01-08-lets-begin-scala-with-gitbucket.md [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+      pulbic Object apply(){
+        return "Hello, world!";
+      }
+    });
+    this.addRoute(route1);
+  }
+}
+```
+
+If you have experience of web application development, you can understand what this code is doing. In Scala, we can write constructor statements in the class definition directory, so DSL such as above example makes easy.
```
Comment (McFoggy) [inline _posts/2017-01-08-lets-begin-scala-with-gitbucket.md]: Not sure I fully understand what you mean in the second sentence (starting from _In Scala_), can you rephrase?

File: _posts/2017-01-08-lets-begin-scala-with-gitbucket.md (hunk 1 above)
Comment (takezoe) [reply _posts/2017-01-08-lets-begin-scala-with-gitbucket.md]: Oops. `param` is correct. The original article had mistake.

File: _posts/2017-01-08-lets-begin-scala-with-gitbucket.md (hunk 2 above)
Comment (takezoe) [reply _posts/2017-01-08-lets-begin-scala-with-gitbucket.md]: I will try to rewrite more clearly.

**Comment by takezoe** [comment]
@McFoggy How do you feel this fix?: af62749c37bd01fcbf1dd7b2e9bd1657ac203fff

**Comment by takezoe** [comment]
@McFoggy Thanks. I adopted yours except `inner class` (replaced with `anonymous class`).
